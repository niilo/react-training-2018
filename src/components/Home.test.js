import React from 'react';
import {BrowserRouter, Route} from 'react-router-dom';
import renderer from 'react-test-renderer';
import Home from './Home';

it('renders default correctly', () => {
  const tree = renderer
    .create(
      <BrowserRouter>
        <Route>
          <Home />
        </Route>
      </BrowserRouter>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
