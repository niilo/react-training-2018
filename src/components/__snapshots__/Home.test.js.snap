// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders default correctly 1`] = `
<div
  className="home-container"
>
  <h1>
    Gihub Battle: Battle your friends:
  </h1>
  <a
    className="button"
    href="/battle"
    onClick={[Function]}
  >
    Battle
  </a>
</div>
`;
